packages/eslint-config-react-union/STYLEGUIDE.md

```diff
@@ -212,7 +219,7 @@ Rule: [*space-before-function-paren*](https://eslint.org/docs/rules/space-before
 
 Severity: error
 
-Other options: `[{"anonymous":"never","named":"never","asyncArrow":"always"}]`
+Other options: `["never"]`
```

Comment:
This will result in a conflict with Prettier, won't it?

Reply:
it does matter because `eslint --fix` is here for help ;) We have it here forever without any problems.

Reply:
I've been having some issues with it, actually. My IDE (VS Code) was not happy with this setting, I had to add the `asyncArrow: always` option. We also have this option set in our internal project.

Reply:
I can see that this change is only regarding the styleguide, not the actual ESLint config, but I believe that we should fix the ESLint config anyway.

Reply:
@wafflepie I will humbly work on it.